src/NuGet.Core/NuGet.ProjectModel/Strings.resx

```diff
@@ -152,4 +152,58 @@
   <data name="MissingVersionOnTool" xml:space="preserve">
     <value>Tools must specify a version range.</value>
   </data>
-</root>
\ No newline at end of file
+  <data name="PackagesLockFile_CentralPackageVersionRemoved" xml:space="preserve">
+    <value>Central package management file doesn't contain version range for {0} package which is specified as CentralTransitive dependency in the lock file.</value>
+  </data>
+  <data name="PackagesLockFile_IncompatibleLockFileVersion" xml:space="preserve">
+    <value>Version specified in the packages.lock.json file is invalid. Current tooling supports version {0} and previous formats.</value>
```

Comment:
It's not invalid. It's incompatible with the current tooling.

Reply:
Changed the message to `Version specified in the packages.lock.json file is incompatible. Current tooling supports version {0} and previous formats.`